src/ui/theme/semanticTokens.ts

```diff
@@ -237,6 +237,9 @@ export const CURRENT_SEMANTIC_TOKENS = {
         },
       },
     },
+    stacksNameAndLogo: {
```

Comment:
Do we need this a semantic token?

Reply:
removed

Reply:
yes because we need to feed the svg a css var. semantic tokens create css vars